util/opentelemetry-util-http/setup.cfg

```diff
@@ -40,3 +40,7 @@ packages=find_namespace:
 
 [options.packages.find]
 where = src
+
+[options.entry_points]
+opentelemetry_instrumentor =
```

Comment:
BTW, is there some integration test that loads the urllib3 or requests instrumentation via the entrypoint, that I could extend to assert on the net.peer.ip attribute being present, so I could verify the wiring is not messed up here?

Reply:
Just to clarify, you are wanting to test if attribute is present if auto-instrumentation is used? If so, I don't believe we have integration tests for those. We currently only have [unit tests](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-instrumentation/tests/test_run.py) for auto-instrumentation.